instance/instance.go

```diff
@@ -118,8 +129,9 @@ func Create(domain string, locale string, apps []string) (*Instance, error) {
 
 	domainURL := config.BuildRelFsURL(domain)
 	i := &Instance{
-		Domain:     domain,
-		StorageURL: domainURL.String(),
+		Domain:        domain,
+		StorageURL:    domainURL.String(),
+		RegisterToken: uniuri.New(),
```

Comment:
i'm not sure we want to rely on external package (through github) to generate secure data